File: data/c++/pattern_8/code.cpp
#include <iostream>
using namespace std;

int main(void)
{
    int num;
    int k = 2;
    cout << "Enter the number of rows and columns for the square: ";
    cin >> num;

    for (int i = 1; i <= num; i++)
    {
        for (int j = 1; j <= num; j++)
        {
            cout << k << " ";
            k += 2;
        }
        cout << endl;
    }

    return 0;
}

File: data/c#/pattern_20/code.cs
using System;
class Grammind
{
    static void Main()
    {
        int num;
        Console.Write("Enter the number of rows and columns for the square: ");
        num = Int32.Parse(Console.ReadLine());

        for (int i = 1; i <= num; i++)
        {
            for (int j = 1; j <= num; j++)
            {
                Console.Write("{0,2:D }", (i * j) % 2);
            }
            Console.Write("\n");
        }
    }
}
